controller.js

```diff
@@ -93,3 +94,21 @@ exports.patchVotes = (req, res, next) => {
   }
   // add else in case its not a number
 };
+
+exports.deleteComment = (req, res, next) => {
+  const { comment_id } = req.params;
+
+  if (!isNaN(Number(comment_id))) {
+    removeComment(comment_id)
+      .then((deletedComment) => {
+        if (deletedComment === "not found") {
+          next("not found");
+        } else {
+          res.status(204).send({});
+        }
+      })
+      .catch((err) => {});
+  } else {
+    next("invalid input");
+  }
```

Comment:
error handling should not be done in this way in the controller. The model should ideally handle all errors and reject them so the .catch can deal with them. You have an empty .catch here which isnt being used for anything.